conda/cli/install.py

```diff
@@ -367,11 +366,11 @@ def install(args, parser, command='install'):
                         else:
                             raise CondaIOError("Can't write the history file", e)
 
-            except RuntimeError as e:
+            except LockError as e:
```

Comment:
I don't think we have LockError used anywhere anymore.  You can probably get rid of it.